_tools/release_docs/api-reference-process.go

```diff
@@ -67,7 +67,7 @@ func stripHTML(filename string) error {
 // replace http://releases.k8s.io/HEAD/docs/xxx.md with
 // http://kubernetes.io/v1.0/docs/xxx.html
 func fixLinks(fileName, outputDir string) error {
-	mdRE := regexp.MustCompile(`http://releases.k8s.io/HEAD/docs(.*?\.)md`)
```

Comment:
What if there was a doc in v1.0 but not in v1.1?
This change will replace http://releases.k8s.io/v1.0/docs/xxx.md with http://kubernetes.io/v1.1/docs/xxx.html which will broken.

This script should replace http://releases.k8s.io/v1.0/docs/xxx.md with http://kubernetes.io/v1.0/docs/xxx.html and http://releases.k8s.io/v1.1/docs/xxx.md with http://kubernetes.io/v1.1/docs/xxx.html (do not change the version)